identity/app/services/ExactTargetService.scala

```diff
@@ -0,0 +1,44 @@
+package services
+
+import com.exacttarget.fuelsdk.{ETResponse, ETSubscriber}
+import utils.SafeLogging
+import common.ExecutionContexts
+import conf.IdentityConfiguration
+import scala.concurrent.Future
+import scalaz.\/
```

Comment:
Is there any way to do without scalaz? 
This is my personal opinion and I know it is already a dependency in frontend but scalaz makes the barrier of entry higher in term of contributing to the code. 
That would be fine if it was a narrow scoped project or if we were using scalaz throughout the whole codebase but we only use it in an handful of places. It is more a burden than anything else at this point I think